lib/keypair.ex

```diff
@@ -123,6 +135,21 @@ defmodule Elixium.KeyPair do
     <<4>> <> x <> y
   end
 
+  defp get_from_private(private) do
+    private
+    |> (fn pkey -> :crypto.generate_key(@algorithm, @curve, pkey) end).()
```

Comment:
This shouldn't be an anonymous function, lets change it to

```elixir
defp get_from_private(private) do
  :crypto.generate_key(@algorithm, @curve, private)
end
```

Reply:
Fixed